cmd/descheduler/app/server.go

```diff
@@ -27,28 +27,30 @@ import (
 	"github.com/spf13/cobra"
 
 	aflag "k8s.io/component-base/cli/flag"
-	"k8s.io/component-base/logs"
 	"k8s.io/klog/v2"
 )
 
 // NewDeschedulerCommand creates a *cobra.Command object with default parameters
 func NewDeschedulerCommand(out io.Writer) *cobra.Command {
-	s := options.NewDeschedulerServer()
+	s, _ := options.NewDeschedulerServer()
 	cmd := &cobra.Command{
 		Use:   "descheduler",
 		Short: "descheduler",
 		Long:  `The descheduler evicts pods which may be bound to less desired nodes`,
 		Run: func(cmd *cobra.Command, args []string) {
-			logs.InitLogs()
-			defer logs.FlushLogs()
+			s.Logs.LogFormat = s.Logging.Format
+			s.Logs.Apply()
+
+			if err := s.Validate(); err != nil {
+				klog.Fatalf("failed validation: %v", err)
```

Comment:
`klog.Fatalf` prints a stack trace. I don't think it is necessary to print a stack trace if CLI option validation fails.

Reply:
changed to use ErrorS.